### PR TITLE
Get Spreadsheet files from Box;

### PR DESCRIPTION
**Pull request by bklukaczewski**
Remove Box API from requirements;
Add celery task for import spreadsheets;

### CODE REVIEW

**Comment by bklukaczewski** [comment]
https://refugeeinfo.atlassian.net/browse/SIS-36

File: balance/settings.py [hunk 1]
```diff
@@ -117,6 +118,9 @@
 
 STATIC_URL = '/static/'
 
+CARD_BALANCE_FILE_URL = 'https://rescue.box.com/shared/static/koh4pt8gqv0xyjvfm9tpdr08lrdgcyef.xls'
```
Comment (kkaczmarczyk) [inline balance/settings.py]: let's make it configurable - regular os.environ.get or this url

File: balance/settings.py [hunk 2]
```diff
@@ -117,6 +118,9 @@
...
+CARD_PHONE_FILE_URL = 'https://rescue.box.com/shared/static/c7lp39hk9a2qf88wmb5b48eydasyy6qp.xlsx'
```
Comment (kkaczmarczyk) [inline balance/settings.py]: same as above - try to make it configurable

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
-boxsdk==1.5.3
 celery==4.0.2
 django==1.10.5
+django-celery-beat==1.0.1
 django-cors-headers==2.0.2
 djangorestframework==3.4.7
+wget==3.2
```
Comment (kkaczmarczyk) [inline requirements.txt]: is this needed? Seems like you are using requests library instead (which sounds good)

**Comment by bklukaczewski** [comment]
@kkaczmarczyk thanks for review! I've fixed those issues and also removed django-celery-beat. Seems it's not needed for now.

File: cards/tasks.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                card.profile_surname = row.get('profile last name')
+                card.profile_id = int(row.get('cardholderid'))
+                card.save()
+            except CardBalance.DoesNotExist:
+                pass
+
+    balance_spreadsheet, cards_spreadsheet = get_spreadsheets()
+    update_cards_with_phone_number(cards_spreadsheet)
+    update_cards_with_balance(balance_spreadsheet)
+    print('import done')
```
Comment (kkaczmarczyk) [inline cards/tasks.py]: this guy seems to stand out.

**Comment by bklukaczewski** [comment]
@kkaczmarczyk done

**Pull request merged**